apps/api/README.md

```diff
@@ -49,3 +49,23 @@ Server ready at http://0.0.0.0:3001
 ```
 
 OpenAPI browser will be available at `http://localhost:3001/docs/`
+
+## Deployment
+
+The sunodo API and postgres database are currently deployed at [fly.io](http://fly.io/).
+As this is a monorepo deployment must be done from the project root directory, and executing the following command:
+
+```shell
+fly deploy --config apps/api/fly.toml --dockerfile ./Dockerfile
```

Comment:
My idea is to have a single Dockerfile at root and adding one stage for each product.
The problem of putting inside `apps/api` is because the context need to be at the root, so I think it's more clear to have the Dockerfile at the context location.
Using `docker build` you can specify the context dir and dockerfile location individually, but I'm not sure if it's possible to specify that way with `fly deploy`